region_file_updater/__init__.py

```diff
@@ -164,6 +167,9 @@ def region_update(source: CommandSource):
 			except Exception as e:
 				msg = '失败，错误信息：{}'.format(str(e))
 				flag = False
+				if str(e)[10:35] == 'No such file or directory' and region_file.split('\\')[0] == 'poi' and os.path.exists(destination):
```

Comment:
I'll suggest to use `isinstance` to test if it's a `FileNotFoundError`. For the removing the file if the source file does not exist part, I'll suggest to make it work with all files instead of poi stuffs since it's correct to sync the "disappeared state" too

Reply:
done :D